app/controllers/ApplicationController.scala

```diff
@@ -313,8 +314,8 @@ class ApplicationController @Inject()(val applicationService: ApplicationService
     implicit request =>
       withApp(appId) { app =>
         def handleValidForm(form: UpdateGrantLengthForm) = {
-          applicationService.updateGrantLength(app.application, GrantLength.from(form.grantLength).get).map { _ =>
-            Redirect(routes.ApplicationController.applicationPage(appId))
+          applicationService.updateGrantLength(app.application, GrantLength.from(form.grantLength.get).get).map { _ =>
```

Comment:
What happen when the form field is none?  .get is usually a bad smell and 2 in a row really looks odd.